### PR TITLE
Multiple of 2019 C++ Solution

### PR DESCRIPTION
**Pull request by maggie-j-liu**

### CODE REVIEW

**Comment by frodakcin** [comment]
@thecodingwizard It seems that if you search this problem in https://usaco.guide/problems/ it says appears in "More on prefix sums" when it actually appears in intro to prefix sums

**Comment by maggie-j-liu** [comment]
According to @thecodingwizard,
> the problem might have been moved. our search implementation doesn't update modules automatically yet oops

so it should be fine...?

**Comment by frodakcin** [comment]
Oh ok

**Comment by jessechoe10** [comment]
Ill write a written edi for this problem a bit later. I don't really like official lmao.

File: solutions/AC Multiple of 2019.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	//initializes array with initializer list
+	//explained here: https://www.learncpp.com/cpp-tutorial/arrays-part-ii/
+	int count[2019]{1};
+	for (int i = n - 1; i >= 0; i--)
+	{
+		//find the remainder of the current number mod 2019
+		num = (num + pow * (s[i] - '0')) % 2019;
+		//increment the count of this remainder
+		count[num]++;
+		pow = (pow * 10) % 2019;
```
Comment (bqi343) [inline solutions/AC Multiple of 2019.mdx:36]: you don't need parentheses here (`*` and `%` have the same precedence)

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Review by HarryWangATX — APPROVED** [review]

**Review by bqi343 — APPROVED** [review]

**Pull request merged**